### ISSUE TITLE
luarocks 3.9.0 causing install issue

### ISSUE THREAD

**Issue by mcsage** [openresty/docker-openresty#204]
Having trouble since luarocks update to install lua modules

```
docker run -ti --rm  openresty/openresty:alpine-fat bash
bash-5.1# luarocks --version
/usr/local/openresty/luajit/bin/luarocks 3.9.0
LuaRocks main command-line interface

bash-5.1# luarocks install lua-resty-openidc
Warning: Failed searching manifest: Failed downloading https://luarocks.org/manifest-5.1 - failed downloading https://luarocks.org/manifest-5.1
Warning: Failed searching manifest: Failed downloading https://raw.githubusercontent.com/rocks-moonscript-org/moonrocks-mirror/master/manifest-5.1 - failed downloading https://raw.githubusercontent.com/rocks-moonscript-org/moonrocks-mirror/master/manifest-5.1
Warning: Failed searching manifest: Failed downloading https://luafr.org/luarocks/manifest-5.1 - failed downloading https://luafr.org/luarocks/manifest-5.1

Error: No results matching query were found for Lua 5.1.
To check if it is available for other Lua versions, use --check-lua-versions.
```

**Comment by neomantra** [comment]
Thanks for the detailed report...   it looks like `luarocks` now has a runtime dependency on `wget`.   I'm adding it to those images now and will do another release.

Something like #125 could help detect this.

```
> docker run -it --rm openresty/openresty:1.19.9.1-11-alpine-fat bash
bash-5.1# luarocks install lua-resty-openidc
Warning: Failed searching manifest: Failed downloading https://luarocks.org/manifest-5.1 - failed downloading https://luarocks.org/manifest-5.1
Warning: Failed searching manifest: Failed downloading https://raw.githubusercontent.com/rocks-moonscript-org/moonrocks-mirror/master/manifest-5.1 - failed downloading https://raw.githubusercontent.com/rocks-moonscript-org/moonrocks-mirror/master/manifest-5.1
Warning: Failed searching manifest: Failed downloading https://luafr.org/luarocks/manifest-5.1 - failed downloading https://luafr.org/luarocks/manifest-5.1

Error: No results matching query were found for Lua 5.1.
To check if it is available for other Lua versions, use --check-lua-versions.

bash-5.1# apk add wget
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/main/aarch64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/community/aarch64/APKINDEX.tar.gz
(1/3) Installing libunistring (0.9.10-r1)
(2/3) Installing libidn2 (2.3.2-r0)
(3/3) Installing wget (1.21.2-r2)
Executing busybox-1.34.1-r5.trigger
OK: 253 MiB in 64 packages

bash-5.1# luarocks install lua-resty-openidc
Installing https://luarocks.org/lua-resty-openidc-1.7.5-1.src.rock
Missing dependencies for lua-resty-openidc 1.7.5-1:
   lua-resty-http >= 0.08 (not installed)
   lua-resty-session >= 2.8 (not installed)
   lua-resty-jwt >= 0.2.0 (not installed)

lua-resty-openidc 1.7.5-1 depends on lua >= 5.1 (5.1-1 provided by VM)
lua-resty-openidc 1.7.5-1 depends on lua-resty-http >= 0.08 (not installed)
Installing https://luarocks.org/lua-resty-http-0.17.0.beta.1-0.src.rock
.....
```

**Comment by neomantra** [comment]
This is released as tag `1.19.9.1-12`.
